gammapy/region/mask.py

```diff
@@ -0,0 +1,33 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.wcs import WCS
+from ..image import exclusion_distance
```

Comment:
In some other place you put the `..image` input in a method.
Should `gammapy.image` be allowed to import `gammapy.region`?
Or the other way around?
Should `ExclusionMask` be located in `gammapy.image`?

(I haven't thought about it much, but at the moment `ExclusionMask` doesn't use region stuff, so maybe move it to `gammapy.image` and at least for now say that `gammapy.region` can import from `gammapy.image`, but we'll try to avoid the other way around?)